src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/rule.py

```diff
@@ -54,5 +54,5 @@ def create_update(self):
             s3_key_path=None,
             account_id=DEPLOYMENT_ACCOUNT_ID,
         )
-        LOGGER.info('Ensuring Stack State for Event Rule forwarding from %s to %s', self.source_account_id, DEPLOYMENT_ACCOUNT_ID)
+        LOGGER.info(f'Ensuring Stack State for Event Rule forwarding from {self.source_account_id} to {DEPLOYMENT_ACCOUNT_ID}')
```

Comment:
Same comment as above on the logger and use of `%s` being preferred over formatted strings with loggers:
```suggestion
        LOGGER.info('Ensuring Stack State for Event Rule forwarding from %s to %s', self.source_account_id, DEPLOYMENT_ACCOUNT_ID)
```